usr/share/rear/lib/config-functions.sh

```diff
@@ -152,18 +124,18 @@ See '$SHARE_DIR/lib/config-functions.sh' for more details."
             # because then scripts in a .../SUSE_LINUX/... sub-directoriy and conf/SUSE_LINUX.conf
             # get sourced twice by the (buggy) SourceStage function in lib/framework-functions.sh
             OS_MASTER_VENDOR="SUSE"
-            # If OS_VERSION is of the form 12.34.56 OS_MASTER_VERSION is only the first part '12'.
-            # Because openSUSE Tumbleweed has rolling releases OS_VERSION is a date of the form YYYYMMDD
-            # so that there is no real OS_MASTER_VERSION which is then the the same as OS_VERSION:
```

Comment:
Please keep the comment about openSUSE Tumbleweed at this place
because it helps me to understand how it behaves for the special
case openSUSE Tumbleweed, e.g. you may adapt the comment to
```
# If OS_VERSION is of the form 12.34.56 OS_MASTER_VERSION is only the first part '12'.
# Because openSUSE Tumbleweed has rolling releases OS_VERSION is a date of the form YYYYMMDD
# so that there is no real OS_MASTER_VERSION which is then the the same as OS_VERSION
# via OS_MASTER_VERSION="${OS_VERSION%%.*}" below.
```

Reply:
Sure!  I thought that I have already resolved that since you already mentioned this concern in https://github.com/rear/rear/pull/3171#issuecomment-2108082864 and the follow-up commit was approved.

Reply:
Oh, sorry, my bad!  I forgot to push those changes after the discussion in #3171 moved off-topic.

Reply:
@jsmeix: Addressed in c7372f7a7adaa27b1ea580e755c6b3745fe01b08.  I have reworded your suggestion a bit, but the main message should be the same.

Reply:
I don't remember all the details that I wrote there
so I may now write different about same things.
But I think
https://github.com/rear/rear/pull/3171#issuecomment-2108082864
is about OS_MASTER_VENDOR versus OS_VENDOR
while the comment about openSUSE Tumbleweed here
is about OS_MASTER_VERSION and OS_VERSION.